index.Rmd

```diff
@@ -50,9 +50,9 @@ d = occ(query= 'Clemmys guttata', from = 'gbif', limit = 1000) %>% occ2df(what =
 ggplot(d,aes(x=longitude,y=latitude))+geom_point() 
   
 ```
-
+ 
 # Results
-The results will show the occurence gbif of spotted turtles based on the citizen science program iNaturalist, and will hopefully shed light on where these creatures are present. Furthermore, the US fish and Wildlife Service National Wetlands Inventory will allow for the understanding of how the current projection of wetlands is correlated or not with the current obsrevation projection of the spotted turtle. 
+The results will show the occurence gbif of spotted turtles based on the citizen science program iNaturalist, and will hopefully shed light on where these creatures are present. Furthermore, the US fish and Wildlife Service National Wetlands Inventory will allow for the understanding of how the current projection of wetlands is correlated or not with the current obsrevation projection of the spotted turtle.  
 
```

Comment:
Good summary of what you will be doing 

Results and conclusion provide good insight on the aims of the project